source/js/main.js

```diff
@@ -42,6 +42,11 @@ document.addEventListener("DOMContentLoaded", function () {
     initializeGA(gaIdentifier);
   }
 
+  // If datalayer is being used, lookout for events.
+  if (window.dataLayer) {
+    initializeCheckForDataLayerEvents();
+  }
```

Comment:
Can you add some context where the initialization of `window.dataLayer` would happen?

Reply:
Hi @tbrlpld ,

This is initialized in the initial GTM script found in [base_master.html](https://github.com/MozillaFoundation/donate-wagtail/blob/efd181d1200dfb6badb34b23ab5f343265e94cd7/donate/templates/base_master.html)

https://github.com/MozillaFoundation/donate-wagtail/blob/efd181d1200dfb6badb34b23ab5f343265e94cd7/donate/templates/base_master.html#L11-L19

This script generates a dataLayer and adds it to the window.

Reply:
Thanks for the pointer @danielfmiranda